services/document-manager/backend/app/docman/resources/document.py

```diff
@@ -16,20 +16,21 @@
 class DocumentListResource(Resource):
```

Comment:
Since tuspy can't add params to the body of a POST, I had to send them in the header (see below). It might be nicer to clean this up so that params **must** come from the header (and not the body or the header).